docs/Configuration.md

```diff
@@ -146,7 +176,41 @@ to specifying a profile JSON file containing the entire
 
 ## ZONEMASTER section
 
-TBD
+The ZONEMASTER section has several keys :
+`max_zonemaster_execution_time`,
+`number_of_professes_for_frontend_testing`,
+`number_of_professes_for_batch_testing`,
+`force_hash_id_use_in_API_starting_from_id`, `lock_on_queue`,
+`maximal_number_of_retries`.
+
+### max_zonemaster_execution_time
+
+This key allows to define the interval in seconds to reload unfinished
+queries.
+This assume that after `max_zonemaster_execution_time`, if the test is
+not finished, it has fail. And it will be run again up to
+`maximal_number_of_retries`.
+
+### maximal_number_of_retries
+
+Specifies the number of time a test is allowed to be run again if
+unfinished after `max_zonemaster_execution_time`.
+
+### number_of_professes_for_frontend_testing
+
+not used in the code
+
+### number_of_professes_for_batch_testing
+
+not used in the code
+
+### force_hash_id_use_in_API_starting_from_id
+
+used -> todo
+
+### lock_on_queue
+
+used -> todo
```

Comment:
@mtoma, can you give a description and legal values for this?

Reply:
It looks like there is a description of the `queue` attribute in the [API documentation](https://github.com/zonemaster/zonemaster-backend/blob/master/docs/API.md#queue)

As I understand it, it is possible to use one database with different Test Agent, so this value works as a label to associate each test to its Test Agent.

Reply:
OK, so then we can possibly refer to the API document, or copy from there.